CMake/ctkMacroBuildQtPlugin.cmake

```diff
@@ -88,7 +88,7 @@ macro(ctkMacroBuildQtPlugin)
   # Wrap
   set(MY_QRC_SRCS "")
   if(CTK_QT_VERSION VERSION_GREATER "4")
-    qt5_wrap_cpp(MY_MOC_CPP ${MY_MOC_SRCS} TARGET ${lib_name} OPTIONS -DHAVE_QT5)
+    qt5_wrap_cpp(MY_MOC_CPP ${MY_MOC_SRCS} OPTIONS -DHAVE_QT5)
```

Comment:
I see, maybe this was related to an older Qt5 version? The Qt 5.2.1 on my Linux machine comes with a Qt5CoreMacros.cmake file where the `qt5_wrap_cpp` macro checks for the TARGET argument and throws an error if TARGET is used together with CMake < 2.8.12.

Correcting my statement above, we only rely on the TARGET argument in the `ctkMacroBuildLib` and `ctkMacroBuildPlugin` macros, but only because we do not call `ctkMacroBuildQtPlugin`. I wonder why you didn't get the same errors the first two macros.